pkg/controller/backup_controller.go

```diff
@@ -479,6 +498,18 @@ func (c *backupController) runBackup(backup *pkgbackup.Request) error {
 	return kerrors.NewAggregate(errs)
 }
 
+func validateUniqueness(backupStore persistence.BackupStore, bucket, name string) []error {
+	var errs []error
+	exists, err := backupStore.BackupExists(bucket, name)
+	if err != nil {
+		errs = append(errs, errors.Errorf("Error checking if backup already exists in object storage: %v", err))
+	}
+	if exists {
```

Comment:
this will only ever be `true` if `err` is `nil`, so I don't think this function really needs to return a slice of errors, could just be a single one.  I would also consider just getting rid of this function and inlining it at L463, since it's just calling `backupStore.BackupExists` and checking the result.